src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+function compareAsString(valueA, valueB) {
+  const stringA = String(valueA);
+  const stringB = String(valueB);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA < stringB) {
+    return -1;
+  } else {
+    return 0;
+  }
```

Comment:
Try to avoid else if and else statements.

if (stringA > stringB) {
    return 1;
} 
if (stringA === stringB) {
   return 0;
}
return -1;

Reply:
@DidenkoDenis306 is right, let's fix this moment and it will be ready to go

Reply:
@DidenkoDenis306 thanks, my mistake